src/app/components/supplied/supplied.component.html

```diff
@@ -0,0 +1,4 @@
+<span *ngIf="editionLevel !== 'diplomatic'" class="{{data.class}}" [ngbPopover]="data?.reason"
```

Comment:
Can this be generalized into a component useful also for surplus?

Reply:
What do you intend to generalize? The popover for the reason?

Reply:
Let's keep it as it is, if more components other than surplus and supplied have the same visualisation, we can add an abstraction of this component